File: python/8 kyu/quarter_of_the_year.py
# Quarter of the year
# https://www.codewars.com/kata/5ce9c1000bab0b001134f5af/train/python

def quarter_of(month):
    if 1 <= month <= 3:
        return 1
    elif 4 <= month <= 6:
        return 2
    elif 7 <= month <= 9:
        return 3
    elif 10 <= month <= 12:
        return 4
    else:
        return 0

File: javascript/8_kyu/quarter_of_the_year.js
// Quarter of the year
// https://www.codewars.com/kata/5ce9c1000bab0b001134f5af/train/javascript

const quarterOf = (month) => {
    if (month <= 3 && month >= 1) {
      return 1;
    } else if (month <= 6 && month >= 4) {
      return 2;
    } else if (month <= 9 && month >= 7) {
      return 3;
    } else if (month <= 12 && month >= 10) {
      return 4;
    }
  }